### ISSUE TITLE
Crash when loading my world

### ISSUE THREAD

**Issue by abdullamu** [Azanor/thaumcraft#1787]
please have look at this 

https://github.com/skyboy/MineFactoryReloaded/issues/446#issuecomment-104992869

**Comment by Anarcraft** [comment]
Confirmed removing Magical Crops beta (4 in my case) resolves the issue.  I've read that version 0.1 works, I'm going to confirm that...

https://github.com/skyboy/MineFactoryReloaded/issues/446#issuecomment-104506645

**Comment by abdullamu** [comment]
for me is very random happens when i update some mods and completely get this error and my worlds stop loading but what i do is update mod or two load again works ,than when new update comes out i update it breaks again so i have to do same thing update something until i get it working . 

same thing happen when installing new mod or removing one so random

**Comment by Anarcraft** [comment]
Yes, it is an intermittent issue.  I had it work 5 times in a row for me (starting up minecraft from the beginning each time) multiple times, other times it failed 5 times in a row.  I've confirmed that Magical Crops 0.1 doesn't produce the issue.

**Comment by BigD999** [comment]
The problem is MFR having an aspect that is added by magic bee's that no in Thaumcraft by default
parseAspects2(Machine.Chronotyper, "3 tempus, 3 bestia, 5 machina, 3 sensus");   
note tempus

**Comment by abdullamu** [comment]
i dont have magic bees installed but still getting this error

**Comment by Anarcraft** [comment]
But removing Magical Crops 4 beta x resolves the issue...

**Comment by NEXUS2345** [comment]
Adding Magic Bees also resolves the issue as well, it may be that Magical Crops is the thing that is causing the crash, but Magic Bees can fix it. So I suggest adding Magic Bees if you don't want to lose your current Magical Crops.

**Comment by Anarcraft** [comment]
I already have Magic Bees installed and that doesn't make a difference, at least in my pack.

**Comment by NEXUS2345** [comment]
It made the difference in ours it seems.

**Comment by Zarkazm** [comment]
From the testing of my issue, link up in the OP, it didn't seem to be any particular mod that set it off on a consistent basis.  For me, if I had Magical Crops Beta 3, I got the error, but for another person they had Beta 3 just fine, but Project E 1.4.7 caused the error for them, where as I was using 1.4.7 successfully.  Then of course there are all the older threads about Witching Gadgets...

At this point, it's either one of these issues:
1.  MFR's Thaumcraft Compat is not integrating with Thaumcraft's API correctly, or
2.  We've discovered a really crazy edge case in Thaumcraft's Aspect system that has always been there, just never found before.

As "users", here is what we know:
1. Issue was originally reported to Azanor (Thaumcraft).
2. Azanor responded that it was an MFR issue. (which at a glance, seemed likely based on the error)
3. Issue reported to skyboy (MFR).
4. skyboy responded that it was a Thaumcraft issue.
5. Issue reported back to Azanor.
6. Azanor has marked the issue as "investigate"

So at this point, the ball is in Azanor's court, but with on going development of Thaumcraft, as well as higher priority bugs on the list, and the fact that this error in particular is so random, this is likely backlogged for awhile.  He might also be holding off on this if he's in the process of updating or rewriting the Aspect system in some way, as it may resolve the issue anyway.

**Comment by BigD999** [comment]
@Zarkazm  do you have magic bees installed

**Comment by Zarkazm** [comment]
@BigD999 Yes.

**Comment by Zarkazm** [comment]
@BigD999 But, during my testing of this issue, Magic Bees was one of the mods I removed/downgraded/readded and didn't change the error in any way

**Comment by BigD999** [comment]
adding magic bees fix the problem for me and I have magic crops B 3 never had a crash since

**Comment by NEXUS2345** [comment]
This is an extremely intermittent crash which makes it very hard to test for, especially since a lot of the time it doesn't actually crash the client, just sends it back to the loading screen. Magic Bees seemed to fix it for us, but then again we have about 128 mods installed, so it is hard to tell which one sometimes.

**Comment by Zarkazm** [comment]
I would just hate to keep pestering Azanor about this issue....I think it's been quantified by now.

**Comment by abdullamu** [comment]
its not about magical bees for me its like balancing issue if i add mod like AOA 2 is didnt work but after updating or adding some other mods or removing some mods it fixes itself .
that why im saying its so weird so random , so for me i have to be careful what to add and what to remove and what to update .
